app/services/user_subscriptions/create.rb

```diff
@@ -0,0 +1,72 @@
+module UserSubscriptions
+  class Create
+    attr_accessor :user, :source_type, :source_id, :subscriber_email, :response
+
+    def self.call(*args)
```

Comment:
Even though it's slightly more verbose, I always spell out the arguments and explicitly forward them. This method is the main interface to this service and `*args` doesn't tell a reader anything, so they have to check the initializer.

Reply:
I agree! My thinking here though is we do it this way (`call(*args)` when using `call` in every other service we use. At this point, it's like an unofficial convention 🤣

Reply:
I may make undoing this convention one of my cooldown projects. Our service objects generally have exactly one public interface method and we know exactly what arguments they need so we can initialize the objects. The lack of clarity for the caller IMHO weighs heavier than the slight redudancy.